src/resources/js/tribe-common.js

```diff
@@ -17,13 +17,32 @@ tribe_auto_sysinfo.ajax = {
 	 */
 	my.init_copy = function () {
 
-		new Clipboard( '.system-info-copy-btn' );
+		var clipboard = new Clipboard( '.system-info-copy-btn' );
+
+		var button_icon = '<span class="dashicons dashicons-clipboard license-btn"></span>';
+
+		var button_text = tribe_system_info.clipboard_btn_text;
 
 		//Prevent Button From Doing Anything Else
 		$( ".system-info-copy-btn" ).click( function ( e ) {
 			e.preventDefault();
 		} );
 
+		clipboard.on('success', function(event) {
```

Comment:
Here are throughout this change - we should have spaces before and after opening parens. So for instance this line should be `clipboard.on( 'success', function( event ) {` and so on for the following lines.